features/align-content-block.yml

```
@@ -0,0 +1,6 @@
+name: align-content in block layouts
+description: The `align-content` property vertically aligns content in block layouts, and no longer requires a flex or grid layout.
```

Optional description suggestion: "no longer" feels like something that will age poorly. It's a good distinction to make here, but maybe we can do something like this that'll feel less like a release notes entry:

```suggestion
description: The `align-content` property vertically aligns content in block layouts, like it does in flex or grid layouts.
```

---

Thanks! I was struggling with that exactly, and your update works great!